Podcast/TabbedPageViewController.swift

```diff
@@ -49,45 +46,40 @@ class TabbedPageViewController: UIViewController, UIPageViewControllerDataSource
     var searchResults: [SearchType: [Any]] = [
         .episodes: [],
         .series: [],
-        .people: [],
-        .tags: []]
+        .people: []]
     
     let pageSize = 20
     var sectionOffsets: [SearchType: Int] = [
         .episodes: 0,
         .series: 0,
-        .people: 0,
-        .tags: 0]
+        .people: 0]
         
     override func viewDidLoad() {
         super.viewDidLoad()
         guard let appDelegate = UIApplication.shared.delegate as? AppDelegate else { return }
-        
         view.backgroundColor = .paleGrey
         automaticallyAdjustsScrollViewInsets = false
         
-        tabBar = UnderlineTabBarView(frame: CGRect(x: 0, y: 64, width: view.frame.width, height: TabBarHeight))
+        tabBar = UnderlineTabBarView(frame: CGRect(x: 0, y: tabBarY, width: view.frame.width, height: tabBarHeight))
         tabBar.setUp(sections: tabSections.map{ type in type.toString() })
         tabBar.delegate = self
         view.addSubview(tabBar)
-            
         tabDelegate = tabBar
-
+        
         viewControllers = SearchTableViewController.buildListOfAllSearchTableViewControllerTypes()
         for viewController in viewControllers {
             guard let searchTableViewController = viewController as? SearchTableViewController else { break }
             searchTableViewController.cellDelegate = self
         }
+        
         pageViewController = UIPageViewController(transitionStyle: .scroll, navigationOrientation: .horizontal, options: nil)
         pageViewController.view.backgroundColor = .offWhite
         let pageVCYOffset: CGFloat = tabBar.frame.maxY + 1 // get a small line between the start of the table view
-        let pageVCHeight = view.frame.height - pageVCYOffset - 44 - 1
+        let pageVCHeight = view.frame.height - pageVCYOffset - appDelegate.tabBarController.tabBarHeight
```

Comment:
We're gonna have to add something so this dynamically changes the VC height when tabBarHeight changes because this is such a custom view controller.

Reply:
Yea this view controller setup is wonky